### DISCUSSION TITLE
Question about the size of the bottom_5mm_acrylic.svg

### DISCUSSION THREAD

**Discussion by mjparme** [ZeroZeroOne-dev/00Key#8] [Q&A] [answered]
I have converted the bottom to an STL and I notice the size is wrong and the holes don't line up with the PCB. My plan was to extrude the SVG into 3d and simply difference it out of an appropriately sized rounded cube in OpenSCAD to get a bottom plate with the holes in the right position. However, I am confused about the holes and size not being right. 

The PCB itself is about 303x168, but the SVG results in an object of 319x185, and the holes don't line up and there aren't enough of them (PCB has 12 holes, bottom.svg has 10 holes). This isn't too big of a deal because I can just measure where the holes are and just position the holes appropriately in OpenScad. But being able to difference out the extruded SVG would be a little quicker. Any ideas why the dimension and hole location isn't correct in the SVG file?  I am going to 3d print a bottom out of clear PETG, should work nicely with the underglow.

Here is a screenshot of the STL file in my slicer that shows dimensions (was just using the slicer to get the dimensions):

<img width="2035" alt="Screen Shot 2022-01-04 at 8 58 13 PM" src="https://user-images.githubusercontent.com/1580996/148154153-321f9b1d-cc7d-4de4-a790-00727947168f.png">

Also, I have made some progress!

![IMG_0865](https://user-images.githubusercontent.com/1580996/148154705-8c7d049f-18ce-4e78-9512-0194821ffaeb.jpg)

**Comment by ZeroZeroOne-dev** [comment, marked as answer]
Hi! Nice progress! looks like you'll have a working board in no time.

The bottom_5mm_acrylic file is for when you want to do the acrylic case design. 
The design is bigger than the pcb itself and only has holes for screwing the acrylic layers together, the pcb is held in place by the switch plate.

Since you want the caseless design you check in the "no case" folder, there are stl files ready for print (with extra glue surface if you wanna do it in 2 halves). I have also uploaded a [full stl](https://github.com/jefjef/00Key/blob/master/no%20case/STL/full.stl) if your printer is big enough.  

Hope this helps!

**Reply by mjparme** [reply]
Ahh, I remember seeing the left/right half STL files in the no case folder before but totally forgot about them. I was heading down the wrong path, thanks for straightening me out. 

Also, thanks for adding that full size STL file. My printer is indeed big enough to print it in one piece (400x300). I actually specifically built this printer so I can print keyboards without splitting. I can print everything except a full size 104-key keyboard on it. 

Here is a Melody96 I printed on it in one piece (bought the PCB from AliExpress then designed the case/plate in OpenSCAD):

![melody96](https://user-images.githubusercontent.com/1580996/148241928-4e8a6657-9760-44ed-a778-c6b94cf61759.jpg)

**Reply by ZeroZeroOne-dev** [reply]
nice melody build!

I used to only have a little ender 3 so thats why i had to split it, a few weeks ago i got a big printer (cr 6 max) also for printing keebs.
